README_CONTAINER_IMAGE.md

```diff
@@ -0,0 +1,37 @@
+# Containerized openshift-ansible to run playbooks
+
+The [Dockerfile](Dockerfile) in this repository uses the [playbook2image](https://github.com/aweiteka/playbook2image) source-to-image base image to containerize `openshift-ansible`. The resulting image can run any of the provided playbooks.
+
+## Build
+
+To build a container image of `openshift-ansible`:
+
+1. Using standalone **Docker**:
+
+        cd openshift-ansible
+        docker build -t certcheck -f images/certificate-check/Dockerfile .
```

Comment:
Rigth, I moved it to the top level dir and forgot to update the doc, will fix, thanks for spotting!